README.md

```diff
@@ -136,7 +140,8 @@ sessions:
 - `host: string` (optional, default: hostname of localhost): Target host of the `tmux` session.
 - `user: string` (optional, default: robot): Target user of the `tmux` session.
 - `wait_for_core: bool` (optional, default: true): Starts the session only after `roscore` is available.
-- `prio: int` (optional, default: 10): Priority of the session. Sessions with the same priority start concurrently. Smaller numbers have higher priority.
+- `prio: int` (optional, default: 10): Priority of starting the session. Sessions with the same priority start concurrently. Smaller numbers have higher priority.
+- `prio_stop: int` (optional, default: `prio`): Priority of stopping the session. Sessions with the same priority stop concurrently. Higher numbers have higher priority.
```

Comment:
why "smaller" number for `prio`, but "higher" numbers for `prio_stop`?

Reply:
The previous logic for start was low -> high values and for stop in reverse high->low. Meaning the last session which was started is the first session which will be stopped. I didn`t change this logic. I just added the function to add a second `prio_stop` value if you need another behavior. (see https://github.com/4am-robotics/robmuxinator/blob/542450e878e2171699dfc9246279c1aeaa140c66/robmuxinator/robmuxinator.py#L687)

Reply:
ah, got it...